### PR TITLE
Milestone 1: Creating data model

### PR DESCRIPTION
**Pull request by Fsher07**
- Created a new app
- Built the project's schema.
- Created and ran the necessary migration files.
- Tables and column names were based on ERD.

### CODE REVIEW

**Review by peterrobert — COMMENTED** [review]
**INVALID 🛑 **

Great work submitting your project for review. However, in order for your project to be reviewed, it needs to have a descriptive readme file. Kindly add a readme file following this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) and  submit your project for another review

File: db/migrate/20220714152933_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
```
Comment (ahmetbozaci) [inline db/migrate/20220714152933_create_users.rb]: - [x] Bio type should be text not string. Kindly change it.

File: db/migrate/20220714153256_create_comments.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :text
```
Comment (ahmetbozaci) [inline db/migrate/20220714153256_create_comments.rb:4]: - [x] Text type should be text not string. Kindly change it.

File: db/migrate/20220714153418_create_posts.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
```
Comment (ahmetbozaci) [inline db/migrate/20220714153418_create_posts.rb:5]: - [ ] Text type should be text not string. Kindly change it.

File: db/migrate/20220714153256_create_comments.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+      t.references :author, index:true, foreign_key: { to_table: :users }
+
```
Comment (ahmetbozaci) [inline db/migrate/20220714153256_create_comments.rb]: - [ ] Kindly add reference also to post model.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @Fsher07  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: db/migrate/20220714153256_create_comments.rb (hunk 2 above)
Comment (mikethreels) [reply db/migrate/20220714153256_create_comments.rb:4]: - [ ] as the previous CR already mentioned please change the type for text to Text not string.

File: db/migrate/20220714153418_create_posts.rb (hunk 3 above)
Comment (mikethreels) [reply db/migrate/20220714153418_create_posts.rb:5]: - [x] as the previous CR already mentioned please change the type for text to Text not string.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Salim, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Fsher07** [comment]
@mikethreels I've already changed them in [db/migrate/20220715134013_sample_name_change_column_type.rb](https://github.com/Fsher07/blog-app-rails/pull/1/files#diff-7c71924001802af0b2cbfe0c2c6f7f75c724eedf4e70486d969cb58301be072b). Thus, you can see the correct types in schema.rb. Should I still make changes in this migration file?I've already changed them in [db/migrate/20220715134013_sample_name_change_column_type.rb](https://github.com/Fsher07/blog-app-rails/pull/1/files#diff-7c71924001802af0b2cbfe0c2c6f7f75c724eedf4e70486d969cb58301be072b). Thus, you can see the correct types in schema.rb. Should I still make changes in this migration file?

**Comment by mikethreels** [comment]
Hi Salim,
My apologies I missed that, it seems to me no other changes are needed. Please send it for review again ☺️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Salim,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Highlights 👍
- schema properly created


<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**